webapp/pages/profile/_id/_slug.vue

```diff
@@ -244,8 +244,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-disabled="$apollo.loading"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-26T09:37:34Z" title="Monday, August 26th 2019, 11:37:34 am +02:00">Aug 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
```


what is the purpose of adding this?
doesn't this default to true anyways?
https://sroze.github.io/ngInfiniteScroll/documentation.html
or am I looking at a different library??

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-26T10:11:42Z" title="Monday, August 26th 2019, 12:11:42 pm +02:00">Aug 26, 2019</time>_

---


Right! I double-checked and it seems it is the default per documentation. I wonder why this fixed some our cypress tests earlier. Maybe another change or the documentation is just wrong.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-26T11:57:05Z" title="Monday, August 26th 2019, 1:57:05 pm +02:00">Aug 26, 2019</time>_

---


Done, ping @mattwr18